CHANGELOG.txt

```diff
@@ -1,4 +1,10 @@
-=== 3.6.0 (unreleased) ===
+
```

Comment:
How did we get the version 3.7? Non eof the changes from 3.5 to 3.7 exist here so we shouldn't mention 3.7 until that time comes to back port changes.